index.html

```diff
@@ -6582,92 +6595,136 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
             <a>AudioWorkletNode</a> instance is destroyed.
           </p>
           <p>
-            When <a>AudioWorkletNode</a>(<var>context</var>, <var>name</var>,
-            <var>options</var>) constructor is called, the user agent must run
-            the following steps <a data-lt="atomic">atomically</a>.
+            Note that the instantiation of these two objects spans the control
+            thread and the rendering thread.
+          </p>
+          <p>
+            When <a>AudioWorkletNode</a>(<var>context</var>,
+            <var>nodeName</var>, <var>options</var>) constructor is invoked,
+            the user agent must perform the following steps on the control
+            thread, where the constructor was called.
           </p>
           <ol>
             <li>Let <var>this</var> be the instance being created by
             constructor of <a>AudioWorkletNode</a> or its subclass.
             </li>
-            <li>If <var>name</var> does not exists as a key in the
-            <a>AudioWorkletGlobalScope</a>’s <a>node name to processor
-            definition map</a>, throw a <code>NotSupportedError</code>
-            exception and abort these steps.
+            <li>If <var>nodeName</var> does not exists as a key in the
+            <a>BaseAudioContext</a>’s <a>node name to parameter descriptor
+            map</a>, throw a <code>NotSupportedError</code> exception and abort
```

Comment:
In any case it seems moot if we are in the main thread and not employing any contexts belonging to other threads. In the past, we had been looking at data belonging to the AWP.